### ISSUE TITLE
Zero-width characters should display guide lines in glyph view to help with anchor placement.

### ISSUE THREAD

**Issue by madig** [trufont/trufont#432]
Title says it all. Extend guide lines over side bearings?

**Comment by madig** [comment]
Alternatively, the width of the Glyphs in question could be set to zero at export time like Glyphs does, but that would require knowledge of the Unicode points somewhere.

**Comment by khaledhosny** [comment]
I’d rather TruFont not second-guess my intention.

**Comment by davelab6** [comment]
You mean if you set an advance width of a glyph that typically had an
advance width of zero, you don't want it to be set to zero at compile time?

**Comment by khaledhosny** [comment]
Yes. There are legitimate ways for having a combining mark with non-zero advance width. Also not all combining marks will be encoded (and UFO does not have way to set glyph classes) so the zeroing might be done inconsistently.
